templates/cloud/public-cloud.html

```diff
@@ -32,7 +32,6 @@ <h1>Ubuntu on public clouds</h1>
 <section class="p-strip is-deep is-bordered">
   <div class="u-fixed-width">
     {% with title="A selection of our certified public cloud partners", json_feed_url="https://partners.ubuntu.com/partners.json?programme__name=Public%20Cloud&featured=true", feed_item_limit=10 %}{% include "shared/_partner-logos.html" %}{% endwith %}
-    <p class="u-align--center"><a href="https://partners.ubuntu.com/find-a-partner?programme=certified-public-cloud">View all certified public cloud partners&nbsp;&rsaquo;</a></p>
```

Comment:
Why was this link removed? It seems to be still there in the copy doc.

Reply:
There's a request to remove it in the comments I think :)